build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -18,4 +18,22 @@
     <StartArguments Condition="'$(StartArguments)' == ''">$(XUnitArguments)</StartArguments>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);AddDefaultTestAppConfig</PrepareForBuildDependsOn>
+  </PropertyGroup>
+
+  <!-- Add a default test app.config, if the project doesn't already have one-->
+  <Target Name="AddDefaultTestAppConfig">
+    <PropertyGroup>
+      <_AlreadyHasAppConfig Condition="'%(None.Filename)%(None.Extension)' == 'app.config'">true</_AlreadyHasAppConfig>
+    </PropertyGroup>
```

Comment:
None is the item, ie `<None Include="App.config" />. This is basically a foreach on all of the None items, with an if check to see if its name is app.config.